flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why limiting this to a single architecture?

Reply:
no support for arm as of now